client/rest-high-level/src/main/java/org/elasticsearch/client/feature/ResetFeaturesResponse.java

```diff
@@ -47,24 +48,30 @@ public static ResetFeaturesResponse parse(XContentParser parser) {
     public static class ResetFeatureStateStatus {
         private final String featureName;
         private final String status;
+        private final Exception exception;
```

Comment:
I cannot comment on the actual line but on line 26, the object parser still uses the "snapshottable" term and is incorrect for this class. Can you address this?